nix/haskell-overrides/HaskellNet-SSL.nix

```diff
@@ -0,0 +1,23 @@
+{ mkDerivation, base, bytestring, connection, data-default
```

Comment:
How are these files created? I assume they're slightly modified variants of the autogenerated ones?

Could we instead use the same overriding mechanism from nixpkgs, like doJailbreak and friends, slightly modifying the autogenerated files, rather than vendoring them in, as we do here?

In any case, that workflow of hasmell-overrides probably needs a small readme in this folder.

Reply:
@akshaymankar says this needs to go away entirely, and be replaced by callCabal2Nix.